### ISSUE TITLE
SimpleElastix docker image size

### ISSUE THREAD

**Issue by sagarr** [romangrothausmann/elastix_scripts#1]
Hi.

Sorry for raising it as an issue, but when I build docker image the size becomes 9.66 GB, Do you know what was your docker image size for elastix when you build it?

**Comment by romangrothausmann** [comment]
No, the builder image is about 1GB and the final image only about 200MB, see:
https://gitlab.com/romangrothausmann/elastix_scripts/container_registry

**Comment by sagarr** [comment]
Thanks for the reply, is there any special docker build command to run.
I ran below command:
`docker build --tag img-reg .`

**Comment by romangrothausmann** [comment]
> Thanks for the reply, is there any special docker build command to run.

Well the GL-CI runs it with stages:
https://github.com/romangrothausmann/elastix_scripts/blob/82af09f1bba0af3115b600065bbb5e2591d25f65/.gitlab-ci.yml#L24-L49
